editor/src/clj/editor/debugging/mobdebug.clj

```diff
@@ -528,7 +527,7 @@
     (assert (= :suspended (-state debug-session)))
     (let [in (.in debug-session)
           out (.out debug-session)]
-      (send-command! out (format "SETB =%s %d" (luajit-path-to-chunk file) line))
+      (send-command! out (format "SETB %s %d" file line))
```

Comment:
mobdebug.lua will strip the @ sign from any source file when the debug hook is triggered (it leaves = alone though). This means that we shouldn't include the @ sign here when we set the breakpoint, otherwise filenames will not match and the debugger will not pause at a breakpoint.

Reply:
Perhaps a good code comment?